aspnet/security/data-protection/compatibility/cookie-sharing.rst

```diff
@@ -10,21 +10,23 @@ Sharing authentication cookies between applications
 
 To share authentication cookies between two different ASP.NET Core applications, configure each application that should share cookies as follows.
 
-1. Add Authentication to your app
+1. In your configure method use the CookieAuthenticationOptions to set up the data protection service for cookies and the AuthenticationScheme to match ASP.NET 4.X
 
 .. code-block:: c#
 
-  public void ConfigureServices(IServiceCollection services)
+  // If using identity
```

Comment:
Instead of distinguishing between these two cases with code comments, let's do it in prose with two separate code blocks.